docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -1,65 +1,28 @@
 
 ## Prerequisites
 
-F# 4.0 or higher must be installed on your machine to use Ionide.
-
-You must also have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` at a command prompt.and pressing **Enter**.
+You must have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` at a command prompt and pressing **Enter**.
 
 ### Windows
 
-If you're on Windows, you have two options for installing F#.
-
-If you've already installed Visual Studio and don't have F#, you can [Install the Visual F# Tools](get-started-visual-studio.md#installing-f).  This will install all the necessary components to write, compile, and execute F# code.
-
-If you prefer not to install Visual Studio, use the following instructions:
-
-1. Install [.NET Framework 4.5 or higher](https://www.microsoft.com/en-US/download/details.aspx?id=30653) if you're running Windows 7.  If you're using Windows 8 or higher, you do not need to do this.
-
-2. Install the Windows SDK for your OS:
-
-    * [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
-    * [Windows 8.1 SDK](https://developer.microsoft.com/windows/downloads/sdk-archive)
-    * [Windows 8 SDK](https://developer.microsoft.com/windows/downloads/sdk-archive)
-    * [Windows 7 SDK](https://www.microsoft.com/download/details.aspx?id=8279)
-
-3. Install the [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159).  You may also need to install [Microsoft Build Tools 2013](https://www.microsoft.com/en-us/download/details.aspx?id=40760).
-
-4. Install the [Visual F# Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48179).
-
-On 64-bit Windows, the compiler and tools are located here:
-
-```
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
-```
-
-On 32-bit Windows, the compiler tools are located here:
-
-```
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
-```
-
-Ionide automatically detects the compiler and tools, but if it doesn't for some reason (for example, the Visual F# Tools were installed to a different directory), you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+If you're on Windows, you must [install Visual Studio with F# support](get-started-visual-studio.md#installing-f).  This will install all the necessary components to write, compile, and execute F# code.
```

Comment:
tabs might come handy in these subsections:
https://review.docs.microsoft.com/en-us/new-hope/resources/tabbed-conceptual?branch=master&tabs=linux%2Cazure-cli#os 

Do you need to add more details here? People might be surprised that in order for them to Get started with F# on VS Code, they also need to install the full VS as well.

will install -> installs

Reply:
Added/addressed